src/apps/code-editor/src/app/components/FileDrawer/FileDrawer.js

```diff
@@ -51,7 +51,7 @@ export const FileDrawer = React.memo(function FileDrawer(props) {
 
         // Logs should always exist
         setLogs(
-          logs.data
+          logs?.data
```

Comment:
If be concerned with the comment on line 52. It seems to think logs would have always existed and rely on that. If they aren't there then `logs?` optional chaining would set undefined which may break other expectations.